bpmn-model/revapi.json

```diff
@@ -1,9 +1,10 @@
 [
   {
     "extension": "revapi.filter",
-    "justification": "Ignore everything not included in the module itself",
+    "id": "filter",
```

Comment:
🔧 The commit message states that it's an explicit choice to not use the `revapi.` prefix for the ids, but it's unclear to me why. Please add it to the commit message

Reply:
It's just because it's shorter yet still descriptive :smile:

Reply:
Haha oh ok, you made it sound like it was an important choice. 😆

Reply:
I've updated the commit